scripts/run_app.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-gunicorn -w 5 -b 0.0.0.0:$1 wsgi
+flask run --host=0.0.0.0 -p $1
```

Comment:
This was a temporary change to use debugger on local, right?

Reply:
I think we can keep this permanently. On paas we define the code that runs using the command var in the manifest, where we pass gunicorn through to run_app_paas. `exec scripts/run_app_paas.sh gunicorn -c /home/vcap/app/gunicorn_config.py wsgi`

We run everything else locally through flask and through gunicorn on paas.